autoRSA.py

```diff
@@ -182,6 +181,13 @@ def argParser(args: list) -> stockOrder:
         else:
             for broker in args[1].split(","):
                 orderObj.set_brokers(nicknames(broker))
+
+        # If next argument is not, set not broker
+        if len(args) > 3 and args[2] == "not":
+            for broker in args[3].split(","):
+                if nicknames(broker) in SUPPORTED_BROKERS:
+                    orderObj.set_notbrokers(nicknames(broker))
+
```

Comment:
Looks good, can you remove the newlines on 184 and 190 to match the rest of the function?